scripts/hipct-distributed.py

```diff
@@ -0,0 +1,97 @@
+import os
+import zarr
+import nibabel as nib
+import numpy as np
+import cv2
+import dask.array as da
+from dask.distributed import Client, LocalCluster
+from skimage import exposure
+from skimage.draw import disk
+
+def setup_dask_cluster():
+    """Initialize a local Dask cluster for distributed computation."""
+    cluster = LocalCluster()
+    client = Client(cluster)
+    print("Dask cluster initialized")
+    return client
+
+def nifti_to_zarr(nifti_path, zarr_path):
+    """Convert a NIfTI file to Zarr format."""
+    img = nib.load(nifti_path)
+    data = img.get_fdata()
+    z = zarr.open(zarr_path, mode='w', shape=data.shape, dtype=data.dtype)
+    z[:] = data
+    print(f"Converted {nifti_path} to {zarr_path}")
+    return z
```

Comment:
Please use @balbasty's tool to create the NIfTI-Zarr file - https://github.com/neuroscales/nifti-zarr/tree/impl/0.3/python

Reply:
I will use this as a guide: https://github.com/neuroscales/nifti-zarr/tree/impl/0.3/python/python. Thank you!